packages/redeem_solver/src/sui_client.ts

```diff
@@ -321,6 +328,41 @@ export class SuiClientImp implements SuiClient {
 		return eventDecoded.event_data.sign_id;
 	}
 
+	async validateSignature(
+		redeemId: number,
+		inputIdx: number,
+		signId: string,
+		nbtcPkg: string,
+		nbtcContract: string,
+	): Promise<void> {
+		await this.ensureIkaInitialized();
+		const tx = new Transaction();
+		const coordinatorId = this.ikaConfig.objects.ikaDWalletCoordinator.objectID;
+
+		tx.moveCall({
+			target: `${nbtcPkg}::nbtc::validate_signature`,
```

Comment:
```suggestion
			target: `${nbtcPkg}::nbtc::record_signature`,
```

Reply:
Can we use a submodule to use nBTC sdk?

Reply:
https://github.com/gonative-cc/workers/issues/254